setup.py

```diff
@@ -0,0 +1,3 @@
+from setuptools import setup
+
+setup()
```

Comment:
Do we really need a `setup.py` file?

Reply:
This is to help folks with backwards compatibility. We don't have to keep it, but it is a nice courtesy.